<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QmitknnInteractiveToolGUI</class>
 <widget class="QWidget" name="QmitknnInteractiveToolGUI">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>241</width>
    <height>402</height>
   </rect>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_3">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>nnInteractive</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <widget class="QPushButton" name="initializeButton">
        <property name="text">
         <string>Initialize</string>
        </property>
        <property name="checkable">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="resetButton">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="text">
         <string>Reset Interactions</string>
        </property>
        <property name="icon">
         <iconset resource="../resources/SegmentationUI.qrc">
          <normaloff>:/Qmitk/nnInteractive/Reset.svg</normaloff>:/Qmitk/nnInteractive/Reset.svg</iconset>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="promptTypeGroupBox">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="title">
      <string>Prompt Type</string>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QPushButton" name="positiveButton">
        <property name="text">
         <string>Positive</string>
        </property>
        <property name="icon">
         <iconset resource="../resources/SegmentationUI.qrc">
          <normaloff>:/Qmitk/nnInteractive/Positive.svg</normaloff>:/Qmitk/nnInteractive/Positive.svg</iconset>
        </property>
        <property name="checkable">
         <bool>true</bool>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="negativeButton">
        <property name="text">
         <string>Negative</string>
        </property>
        <property name="icon">
         <iconset resource="../resources/SegmentationUI.qrc">
          <normaloff>:/Qmitk/nnInteractive/Negative.svg</normaloff>:/Qmitk/nnInteractive/Negative.svg</iconset>
        </property>
        <property name="checkable">
         <bool>true</bool>
        </property>
        <property name="checked">
         <bool>false</bool>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="interactionToolsGroupBox">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="title">
      <string>Interaction Tools</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QPushButton" name="pointButton">
        <property name="text">
         <string>Point</string>
        </property>
        <property name="iconSize">
         <size>
          <width>24</width>
          <height>24</height>
         </size>
        </property>
        <property name="checkable">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="boxButton">
        <property name="text">
         <string>Box</string>
        </property>
        <property name="iconSize">
         <size>
          <width>24</width>
          <height>24</height>
         </size>
        </property>
        <property name="checkable">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="scribbleButton">
        <property name="text">
         <string>Scribble</string>
        </property>
        <property name="iconSize">
         <size>
          <width>24</width>
          <height>24</height>
         </size>
        </property>
        <property name="checkable">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="lassoButton">
        <property name="text">
         <string>Lasso</string>
        </property>
        <property name="iconSize">
         <size>
          <width>24</width>
          <height>24</height>
         </size>
        </property>
        <property name="checkable">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="maskButton">
        <property name="text">
         <string>Initialize with Mask</string>
        </property>
        <property name="icon">
         <iconset resource="../resources/SegmentationUI.qrc">
          <normaloff>:/Qmitk/nnInteractive/Mask.svg</normaloff>:/Qmitk/nnInteractive/Mask.svg</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>24</width>
          <height>24</height>
         </size>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="autoRefineCheckBox">
        <property name="text">
         <string>Auto-refine mask</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="autoZoomCheckBox">
        <property name="text">
         <string>Auto-zoom</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../resources/SegmentationUI.qrc"/>
 </resources>
 <connections/>
</ui>
